### DISCUSSION TITLE
Copilot is encountering temporary issues

### DISCUSSION THREAD

**Discussion by valentinbeggi** [community/community#32298] [Copilot Conversations] [answered]
Hello,

Since this morning my Copilot is down and I can't get it working again.
Was using Copilot Nightly extension, deinstalled it and installed Copilot classic. I relaunched VSCode and also my laptop but nothing seems to work.
Could anyone help me ?
Thanks

**Comment by iamcjwilliams** [comment]
Hello @valentinbeggi, We're sorry for the trouble here. Our Copilot team is working to improve the quality of the extension. Please visit this discussion to get the latest version: https://github.com/orgs/community/discussions/30716

**Reply by valentinbeggi** [reply]
Hey @iamcjwilliams thanks for your reply
I'm using version v1.44.6735 , but it still doesn't work 🙁

**Comment by davecheney** [comment, marked as answer]
@valentinbeggi please go to your Copilot extension `settings.json` and remove the line that says 
```
 "debug.overrideEngine": "copilot-logit"
```

**Reply by davecheney** [reply]
@laindream @EmanuelAmico ☝ FYI

**Reply by davecheney** [reply]
@PeterAVG ☝️  FYI

**Reply by valentinbeggi** [reply]
Yep it worked, thanks a lot ☀️

**Reply by davecheney** [reply]
@aidantorrence ☝️ FYI
